src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,70 @@
+
+However, this practice may cause problems for people with Dyslexia. It can make some letters, numbers, punctuation, and other characters become indistinguishable. This can make it easy to confuse one character for another. As a result, people with Dyslexia may prefer fonts where all the characters are unique. 
+
+
+## The questionable utility of Dyslexic fonts?
+
+Dyslexia fonts are typefaces designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. They are intended to help people with Dyslexia.
+
+According to recent studies, it has been proven that Dyslexia fonts do **not** help people **read faster and make fewer mistakes**. In fact, most people with Dyslexia prefer Comic Sans, Verdana, and Helvetica to other fonts. For a detailed  review of the research, see [Good Fonts for Dyslexia (PDF)](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf)
+
+
+## General font tips
+
+The following are general tips that improve the overall quality of your website's typography:
+
+- Font size should be at least 12-14pt or its equivalent. This help people with cognitive disabilities, language and learning disabilities, as well as people with low vision who may not be able to perceive the text.  
+- Larger line spacing improves readability. Proper line spacing makes [web content](https://en.wikipedia.org/wiki/Web_content) flow smoothly, and ensures that the visually rendered text is presented in such a manner that is perceived without its layout interfering with its readability. According to the [WCAGs Visual Presentation(level AAA) success criterion](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-visual-presentation.html), Line height (line spacing) should be at least 1.5 times the font size. 
+
+**Note**: There are no hard rules that defines when the line spacing is too loose. However, when the lines of text seem to be visually floating from each other, it is a sign of excess line spacing.
+
+- Use a small number of fonts, ideally 1 or 2. This is because multiple fonts makes a website look unstructured and unprofessional. Also, multiple fonts obstructs the flow of content of the web page as people spend time adjusting to each new font. Hence, reducing readability. 
+- Use **bold** to add emphasis rather than *italics*. This is because large amounts of italicized text can be difficult to read. 
+- Avoid large amounts of UPPERCASE text. When words are written in all caps, they have a uniform rectangular shape, meaning some readers may not be able to identify words by their shape.
+- Animated text must be used sparingly and honor [reduced motion preferences](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion). This is because animations can be distracting, especially for a person with [AD/HD](https://www.cdc.gov/ncbddd/adhd/facts.html).
```

Comment:
```suggestion
- Animated text must be used sparingly and honor [reduced motion preferences](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion). This is because animations can be distracting, especially for a person with [ADHD](https://www.cdc.gov/ncbddd/adhd/facts.html).
```